tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestCLDRLocaleCoverage.java

```diff
@@ -393,16 +393,24 @@ public void TestParentCoverage() {
                     if (parent == null || parent.equals(LocaleNames.ROOT)) {
                         break;
                     }
-                    if (!defaultContentLocales.contains(parent) && !parent.equals("en_001")) { // en_001 is generated later from en_GB
```

Comment:
The changes in formatting make it hard to see what the **real** change was. How did the original of this file escape being formatted by spotless?

Reply:
it didn't escape, it was formatted 'correctly'

The main change is just adding a defaultContent check. The rest is just test output changes.